hnn_core/parallel_backends.py

```diff
@@ -56,10 +51,9 @@ def _gather_trial_data(sim_data, net, n_trials):
         spikedata = sim_data[idx][1]
         net.spikes._spike_times.append(spikedata[0])
         net.spikes._spike_gids.append(spikedata[1])
-        net.gid_dict = spikedata[2]  # only have one gid_dict
-        net.spikes.update_types(net.gid_dict)
+        net.gid_ranges = spikedata[2]  # only have one gid_ranges
+        net.spikes.update_types(net.gid_ranges)
         net.spikes._vsoma.append(spikedata[3])
-        net.spikes._times.append(net.times.tolist())
```

Comment:
Is there any reason different trials should be allowed to have different `times`?!

Reply:
I'm guessing that the goal here was to maintain consistent formatting between the primary attributes of the `Spikes` class, but you're right that this is probably unnecessary. Any considerations we're missing @ntolley?

Reply:
Consistency was the main motivation, the way it is written should not allow for different `times` across trials. `times` is actually created from directly from `params` when the network is instantiated, this function is really just copying the data.

Not really a big deal to change and probably cleaner this way. Most important thing is for the examples to demonstrate how to use the variable in plotting.